src/screens/Home/index.tsx

```diff
@@ -1,8 +1,20 @@
 /* eslint-disable react/react-in-jsx-scope */
+import { useQuery } from 'react-query';
+
+import { getBooks } from 'services/BooksService';
+
+import BookList from '../../components/BookList';
+
 import styles from './styles.module.scss';
 
 function Home() {
-  return <div className={styles.app}>Home</div>;
+  const { isLoading, data } = useQuery('books', getBooks);
```

Comment:
please can you use error
```suggestion
  <div className={styles.app}>
      {isLoading ? <span className={styles.loading}>Loading...</span> : error ? <div>Intenta más tarde </div><BookList books={data} />}
    </div>
```